packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -56,6 +56,20 @@ import MQAProcessor from '../metrics/mqa-processor';
 
 import InMeetingActions from './in-meeting-actions';
 
+const logRequest = (request, {header = '', success = '', failure = ''}) => {
+  LoggerProxy.logger.log(header);
+
+  return request
+    .then((...args) => {
+      LoggerProxy.logger.log(success);
+
+      return args;
+    })
+    .catch((error) => {
+      LoggerProxy.logger.error(failure, error);
+      throw error;
+    });
+};
```

Comment:
I like this method, would be cool to consider this as an improvement towards updating our request dependency.

Reply:
Not sure if a short JSDoc is needed here.